src/Store/Index.js

```diff
@@ -1,12 +1,12 @@
 import Vuex from 'vuex'
 import Vue from 'vue'
-import todos from './modules/todos'
+import todos from './Modules/todos'
```

Comment:
Paths are case-sensitive in some platforms, and not in others. So, it is better to use proper cases in the code.